sites/default/redirects.php

```diff
@@ -110,10 +110,10 @@ function _lexky_get_legacy_document_redirect($incoming_path) {
     return 'http://previous.lexingtonky.gov' . str_replace('?', '@', $incoming_path);
   }
```

Comment:
Do we still need this file? I would believe not, since we are no longer handling redirects outside of configuration from the modules.

Reply:
I'll say, for this PR, let's leave the file. It's doing more things than redirect for certain urls. But, anything referencing the redirect file, should be able to be removed. So _lexky_get_redirects shouldn't be needed. Everything else we should keep and come by to the necessity of them later.  @ChrispyAPAX

Reply:
We should also be getting rid of the redirects.json. We aren't using it anymore